setup.py

```diff
@@ -65,7 +65,7 @@
                               extra_link_args=EXTRA_LINK_ARGS)],
     "install_requires": ["screed>=0.9", "PyYAML>=3.11"],
     "extras_require": {
-        'test' : ['pytest', 'pytest-cov', 'numpy', 'matplotlib', 'scipy'],
+        'test' : ['pytest', 'pytest-cov', 'numpy', 'matplotlib', 'scipy', 'ijson'],
```

Comment:
You are right. This was originally implemented as a very optional JSON format, but it now going to be the possible default.